--- oapr-0.1/oapu/Makefile.am.orig	2004-01-20 12:00:11.000000000 +0100
+++ oapr-0.1/oapu/Makefile.am	2004-02-07 20:25:06.060091416 +0100
@@ -23,4 +23,4 @@
 	oapr_logstream.cpp
 
 liboapu_la_LDFLAGS = @apu_ldflags@
-liboapu_la_LIBADD = @apu_ldlink@ @apu_libs@
+liboapu_la_LIBADD = @apu_ldlink@ @apu_libs@ ../oapr/liboapr.la
--- oapr-0.1/oapx/Makefile.am.orig	2004-01-20 12:00:57.000000000 +0100
+++ oapr-0.1/oapx/Makefile.am	2004-02-07 20:25:37.617293992 +0100
@@ -34,4 +34,4 @@
                      oapr_config.cpp \
                      oapr_financial.cpp
 liboapx_la_LDFLAGS = @apr_ldflags@
-liboapx_la_LIBADD = @apr_ldlink@ @apr_libs@
+liboapx_la_LIBADD = @apr_ldlink@ @apr_libs@ ../oapr/liboapr.la
